pyspark_ai/prompt.py

```diff
@@ -84,15 +98,29 @@
 ]
 
 SPARK_SQL_SUFFIX = """\nQuestion: Given a Spark temp view `{view_name}` {comment}.
-It contains the following columns:
+The dataframe contains the column names and types in this format:
+column_name: type.
+It's very important to ONLY use the verbatim column names in your resulting SQL query.
+For example, the verbatim column name from the line 'Fruit Color: string' is 'Fruit Color'.
+So, a resulting SQL query would be: SELECT * FROM view WHERE 'Fruit Color' = 'orange'.
```

Comment:
shall we use backtick instead of single quote? Single quote means string literal in Spark SQL